### PR TITLE
Allow user to select on position on map without locating themselves

### PR DESCRIPTION
**Pull request by Shriukan33**
We used to ask for location permission when the user clicked on the "Je partage ma position" button. This was not ideal because the user couldn't use the map to set their location without giving us permission to track them.

The map is now centered on Nantes unzoomed by default, and user is now only located once they click the control button to locate themselves on the map.

### CODE REVIEW

**Review by JeffAbrahamson — APPROVED** [review]
Please add an issue number before merging.

There's an ongoing bug that the map is not shown (and there's still a button offering to localise).  And that the arrow button is modal, which is thoroughly bizarre.  For example, if I click the arrow to localise myself, it gets it wrong, I click the map, I've now localised myself correctly, a second later the map jumps me back to the wrong spot.

**Comment by Shriukan33** [comment]
I've fixed the modal behavior and "random teleport". 
Also I changed the icon to be more what I would expect from a button that brings me to my location.

**Comment by JeffAbrahamson** [comment]
First commit comment:

--- 
Allow user **to** point to a position on **the** map ~~without giving~~**to provide** device location
    
We used to localise the user on clicking the
"Je partage ma position" button. This was not ideal because
it introduced an unuseful extra step (revealing the map)

and because the user should be able to position himself
on the map without triggering geolocalisation.
    
With this commit the map is centred on Nantes at wide
(departement) zoom.  The user geolocalises only on clicking
the location control.
    
**P**art of #932**.**

--- 

Note that _we_ did not ask for permission.  That is the browser.

**Comment by JeffAbrahamson** [comment]
LGTM, modulo the commit message.

In the second commit, you're missing a period at EOL on the last line.  And it has an extra space.

**Pull request merged**
